specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -55,6 +64,8 @@ export function Notifications({
       const queryString = since
         ? `?since=${encodeURIComponent(since.toISOString())}`
         : '';
+      lastFetchedTimestamp.current = since;
```

Comment:
hmm, you set `lastFetchedTimestamp.current` to `since`, and then you call `doFetch` again with the value of `lastFetchedTimestamp.current` - but you never call `new Date()` to get a new timestamp???